src/Synergism.ts

```diff
@@ -1281,6 +1282,12 @@ const loadSynergy = () => {
         if (!player.autoResearchToggle) {
             document.getElementById("toggleautoresearch").textContent = "Automatic: OFF"
         }
+        if (player.autoResearchMode === 'cheapest') {
+            document.getElementById("toggleautoresearchmode").textContent = "Automatic mode: Cheapest"
+        }
+        if (player.autoResearchMode === 'manual') {
```

Comment:
```suggestion
        } else if (player.autoResearchMode === 'manual') {
```

Reply:
I agree it's cleaner, but I've followed the style of every other if surrounding it, it's probably best to leave it as is for the sake of consistency

Reply:
the code is rather messy and continuing bad styling isn't a good idea